docs/spec/errors/error.json

```diff
@@ -46,7 +46,8 @@
                 },
                 "culprit": {
                     "description": "Function call which was the primary perpetrator of this event.",
-                    "type": ["string", "null"]
+                    "type": ["string", "null"],
+                    "maxLength": 1024
```

Comment:
This is a breaking change.

Reply:
whatever you both prefer, but imo this is ok - it is very unlikely that this causes issues in practical terms, considering the big picture

Reply:
I suggest to at least check with the agent devs if this could be a problem (e.g. verbose class and method names in java) etc.

Reply:
https://github.com/elastic/apm/issues/43 opened for discussion